db/DB.js

```diff
@@ -36,9 +37,9 @@ const DB = {
       .where("email", "=", email);
   },
 
-  async addSubscriberUnverifiedEmailHash(email) {
+  async addSubscriberUnverifiedEmailHash(email, fxNewsletter = false) {
     const res = await knex("subscribers").insert(
-      { email: email, sha1: getSha1(email), verification_token: uuidv4(), verified: false }
+      { email: email, sha1: getSha1(email), verification_token: uuidv4(), verified: false, fx_newsletter: fxNewsletter }
```

Comment:
nit: this line may be starting to be grown up enough to warrant multiple lines for readability.